tests/LibrdkafkaTestApp.py

```diff
@@ -66,10 +67,36 @@ def __init__(self, cluster, version, conf=None,
 
             elif mech == 'OAUTHBEARER':
                 self.security_protocol = 'SASL_PLAINTEXT'
-                conf_blob.append('enable.sasl.oauthbearer.unsecure.jwt=true\n')
-                conf_blob.append(
-                    'sasl.oauthbearer.config=%s\n' %
-                    self.conf.get('sasl_oauthbearer_config'))
+                oidc = cluster.find_app(OauthbearerOIDCApp)
+                if oidc is not None:
+                    conf_blob.append('sasl.oauthbearer.method=%s\n' %
+                                     oidc.conf.get('sasl_oauthbearer_method'))
+                    conf_blob.append('sasl.oauthbearer.client.id=%s\n' %
+                                     oidc.conf.get(
+                                         'sasl_oauthbearer_client_id'))
```

Comment:
Where are these set up? (should be the OIDC..App in trivup, but I can't see them there?

Reply:
I updated the trivup PR with the latest set up.